packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4930,7 +4930,7 @@ export default class Meeting extends StatelessWebexPlugin {
         this.reconnect({networkDisconnect: true});
         NewMetrics.submitClientEvent({
           name: 'client.ice.end',
-          payload: {canProceed: false},
+          payload: {canProceed: false, icePhase: 'JOIN_MEETING_RETRY'},
```

Comment:
this code is executed when we lose ICE connection in-meeting, after we initially established it successfully during join, so the icePhase should be 'IN_MEETING'